src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -98,33 +98,30 @@ public void Deconstruct(out int a, out string b)
             comp.VerifyDiagnostics();
             comp.VerifyIL("C.Main", @"
 {
-  // Code size       48 (0x30)
+  // Code size       44 (0x2c)
   .maxstack  3
   .locals init (long V_0, //x
                 string V_1, //y
                 int V_2,
-                string V_3,
-                string V_4)
+                string V_3)
   IL_0000:  newobj     ""C..ctor()""
   IL_0005:  ldloca.s   V_2
   IL_0007:  ldloca.s   V_3
-  IL_0009:  call       ""void C.Deconstruct(out int, out string)""
+  IL_0009:  callvirt   ""void C.Deconstruct(out int, out string)""
```

Comment:
I'm not sure why this changed from `call` to `callvirt`. I'll check with Vlad on whether this is significant.